src/test/java/nl/tudelft/contextproject/audio/AudioTestUtil.java

```diff
@@ -0,0 +1,48 @@
+package nl.tudelft.contextproject.audio;
```

Comment:
The AudioTestUtil does not test the AudioUtil, but is a utility class for testing with support for audio without failing on travis. Therefore, it is better suited where it is.